base_classes/NXtransformations.nxdl.xml

```diff
@@ -108,6 +126,143 @@
+		        phi=0
+		          @depends_on=chi
+		          @transformation_type=rotation
+		          @vector=[-1 -0.0037 -0.002]
+		          @units=degrees
+		        chi=0
+		          @depends_on=sam_x
+		          @transformation_type=rotation
+		          @vector=[0.0046 0.0372 0.9993]
+		          @units=degrees
+		        sam_x=0
+		          @depends_on=sam_y
+		          @transformation_type=translation
+		          @vector=[1 0 0]
+		          @units=mm
+		        sam_y=0
+		          @depends_on=sam_z
+		          @transformation_type=translation
+		          @vector=[0 1 0]
+		          @units=mm
+		        sam_z=0
+		          @depends_on=omega
+		          @transformation_type=translation
+		          @vector=[0 0 1]
+		          @units=mm
+		        omega=174
+		          @depends_on=.
+		          @transformation_type=rotation
+		          @vector=[-1 0 0]
+		          @units=degrees
+
+		**Example 2: different coordinate system**
+
+		Define a laboratory reference frame with the X-axis along the beam and the Z-axis opposite to the direction of gravity.
+		Three point detectors are positioned in this reference:
+
+		* *transmission*:
+			* point detector in the beam
+			* 20 cm downstream from the sample (the origin of the reference frame)
+		* *vertical*:
+			* point detector 10 cm downstream from the sample
+			* making an angle of 5 degrees with the beam w.r.t. the sample
+			* positioned in the XZ-plane above the XY-plane
+		* *horizontal*:
+			* point detector 11 cm downstream from the sample
+			* making an angle of 6 degrees with the beam w.r.t. the sample
+			* positioned in the XY-plane above the XZ-plane
+
+		The coordinates of the point detectors in the laboratory reference frame are
+
+		* *transmission*: :math:`X_\text{lab} = T_x(20) . X_d`
+		* *vertical*: :math:`X_\text{lab} = R_y(-5) . T_x(10) . X_d`
+		* *horizontal*: :math:`X_\text{lab} = R_x(-90) . R_y(-6) . T_x(11) . X_d`
+
+		where
+
+		* :math:`T_x`, :math:`T_y`, :math:`T_z`: active transformation matrices for translation along the X, Y and Z axes
+		* :math:`R_x`, :math:`R_y`, :math:`R_z`: active transformation matrices for rotation around the X, Y and Z axes
+		* :math:`X_d` is a coordinate in the detector reference frame.
+
+		Note that as these are point detectors, we only have one coordinate :math:`X_d=[0,0,0,1]^T`.
+
+		.. code-block::
+
+		  entry:NXentry
+		    instrument:NXinstrument
+		      vertical:NXdetector
+		        depends_on=position/distance
+		        position:NXtransformations
+		          distance=10            # move downstream from the sample
+		            @depends_on=polar
+		            @units=cm
+		            @vector=[1 0 0]
+		          polar=5                # title above the horizontal plane
+		            @depends_on=azimuth
+		            @units=degrees
+		            @vector=[0 -1 0]
+		          azimuth=0              # stay in the vertical plane
+		            @depends_on=/entry/coordinate_system/beam
```

Comment:
This should be `@depends_on="."` as it attached to the laboratory frame's origin.